crawler/src/main/java/com/minibcr/crawler/CrawlerApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.minibcr.crawler;
+
+import java.io.IOException;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication public class CrawlerApplication {
```

Comment:
maybe put the annotation above the class declaration as we usually do? Ik this is probably auto formatting but its good to follow one style